src/components/carousel/_index.scss

```diff
@@ -0,0 +1,159 @@
+@use "../../scss";
+@use "~slick-carousel/slick/slick";
+
+.c-carousel {
+	@include scss.component-properties("carousel");
+
+	/* Arrows */
+	.slick-prev,
+	.slick-next {
+		cursor: pointer;
+		font-size: 0;
+		line-height: 0;
+		position: absolute;
+		top: 50%;
+		display: block;
+
+		&:hover {
+			color: transparent;
+			background: transparent;
+
+			&:before {
+				@include scss.component-properties("carousel-buttons-hover-before");
+			}
+		}
+
+		&:focus {
+			color: transparent;
+			background: transparent;
+
+			&:before {
+				@include scss.component-properties("carousel-buttons-focus-before");
+			}
+		}
+
+		@include scss.component-with-states("carousel-buttons", [hover, focus]);
+	}
+
+	.slick-next {
+		&:before {
+			font-family: "Arial", serif;
+			content: ">";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+		}
+
+		@include scss.component-with-states("carousel-button-next", [before]);
+	}
+
+	.slick-prev {
+		&:before {
+			font-family: "Arial", serif;
+			content: "<";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+		}
+
+		@include scss.component-with-states("carousel-button-previous", [before]);
+	}
+
+	.slick-prev.slick-disabled {
+		&:before {
+			opacity: 0.25;
+		}
+
+		@include scss.component-with-states("carousel-button-previous-disabled", [before]);
```

Comment:
Just curious about semantics here.  I missed this before.  Is `.slick-disabled` denoting that the carousel is disabled or the 'previous' is disabled?  That could determine the order of the theme property token words.

`carousel-button-previous-disabled` vs `carousel-disabled-button-previous`

Same question is posed for 'next' below.